scripts/install.sh

```diff
@@ -142,9 +129,10 @@ sudo apt -y install \
   libcurl4-openssl-dev \
   acl
 
-git clone https://github.com/tpm2-software/tpm2-tss.git $INSTALL_DIR/tpm2-tss
-cd $INSTALL_DIR/tpm2-tss
-./bootstrap
+cd $INSTALL_DIR
+wget https://github.com/tpm2-software/tpm2-tss/releases/download/3.1.0/tpm2-tss-3.1.0.tar.gz
+tar -xf tpm2-tss-3.1.0.tar.gz && rm tpm2-tss-3.1.0.tar.gz
```

Comment:
I think you can directly extract the archive into a given directory.
This will avoid `mv tpm2-tss-3.1.0 tpm2-tss`